src/thor/triplegbuilder.cc

```diff
@@ -659,10 +659,15 @@ void AddTripIntersectingEdge(const AttributesController& controller,
   // Set the sign info for the intersecting edge if requested
   if (controller.attributes.at(kNodeIntersectingEdgeSignInfo)) {
     if (intersecting_de->sign()) {
-      GraphId beginnode = graphreader.GetBeginNodeId(intersecting_de, graphtile);
-      valhalla::baldr::graph_tile_ptr t2 = graphreader.GetGraphTile(beginnode);
-      size_t edge_idx = intersecting_de - t2->directededge(0);
-      std::vector<SignInfo> edge_signs = t2->GetSigns(edge_idx);
+      // To retrieve the sign information from the intersecting_de I need
+      // to access the graph-tile for the "begin node" of the intersecting_de.
+      // GraphReader::GetGraphTile(DirectedEdge *) returns the graph-tile for
+      // the "end node". Hence, I need to obtain the opposing-directed-edge
+      // and then ask for it's graph tile. (Sigh)
+      GraphId opp_intersecting_de = graphtile->GetOpposingEdgeId(intersecting_de);
```

Comment:
this call call is actually very dangerous and i kind of think we should remove it! it assumes that the `endnode` of `intersecting_de` is in the tile you are calling it on. if `intersecting_de` is an edge that ends in another tile, this will get the wrong node and get the wrong opposing edge id: https://github.com/valhalla/valhalla/blob/master/valhalla/baldr/graphtile.h#L250

if the edge doesnt leave the tile it would work. also nitpick naming this with "_de" is confusing because its a graphid not a "de"